src/caselawclient/models/utilities/move.py

```diff
@@ -30,7 +31,7 @@ def update_document_uri(
     :param api_client: An instance of MarklogicApiClient used to make the search request
     :return: The URL associated with the `target_citation`
     """
-    new_ncn_based_uri = caselawutils.neutral_url(target_citation.strip())
+    new_ncn_based_uri = caselawutils.neutral_url(NeutralCitationString(target_citation.strip()))
```

Comment:
I think we could push this up into the method signature, so `target_citation` has to be a `NeutralCitationString`. That way it'll force stuff downstream to also use that type.

Reply:
See some changes in https://github.com/nationalarchives/ds-caselaw-custom-api-client/pull/705/commits/0876d66894885658e86b2f0528d418ff11d269a7 if you want to just lift them – my only "don't do this" would be typing `api_client`, because it forces us to fix a load of annotation stuff.

Which we should, but I don't want to hold everything else up for it.